src/components/CardContainer/CardContainer.js

```diff
@@ -45,7 +61,11 @@ class CardContainer extends React.Component {
                             key={dog.id}
                             name={dog.name}
                             height={dog.height.metric + "cm"}
-                            temperament={dog.temperament}/>
+                            temperament={dog.temperament}
+                            id={dog._id}
+                            clickEvent={(id)=>{
```

Comment:
can we have spaces around the `=>` and lose the brackets around `id`

Reply:
On it

Reply:
On it